templates/volontulo/statics/site-news.html

```diff
@@ -0,0 +1,16 @@
+{% load bootstrap3 %}
+
+{% bootstrap_css %}
+
+<div class="media-body">
+    <div class="media-left">
+        21 października 2015
+    </div>
+    <div class="media-body">
+        <h3 class="media-heading">News o działaniu portalu</h3>
```

Comment:
Using headings to styling is not a good idea. Probably desirable situation is h1 with "News page title" and h2 for every media-heading. In HTML5 there is a something like 'syllabus' or 'synopsis', which allow building table of content using properly embedding of headings elements.